kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableWritesSuite.scala

```diff
@@ -849,6 +978,29 @@ class DeltaTableWritesSuite extends DeltaTableWriteSuiteBase with ParquetSuiteBa
     }
   }
 
+  private def getMetadataActionFromCommit(
+    engine: Engine, table: Table, version: Long): Optional[Metadata] = {
+    val logPath = new Path(table.getPath(engine), "_delta_log")
+    val file = engine
+      .getFileSystemClient
+      .listFrom(FileNames.listingPrefix(logPath, version)).next
```

Comment:
you can directly construct the file status using `FileNames.deltaFile` method.

Reply:
Seems `FileNames.deltaFile` method can only return the String?